lib/goliath/constants.rb

```diff
@@ -5,51 +5,52 @@ module Constants
     # Force external_encoding of request's body to ASCII_8BIT
     INITIAL_BODY.encode!(Encoding::ASCII_8BIT) if INITIAL_BODY.respond_to?(:encode)
 
-    SERVER_SOFTWARE = 'SERVER_SOFTWARE'
-    SERVER          = 'Goliath'
+    SERVER_SOFTWARE   = 'SERVER_SOFTWARE'
```

Comment:
Can we revert these chunks to original formatting? I want to avoid auto-formatter wars.. :)